bundles/org.openhab.binding.melcloud/src/main/resources/ESH-INF/thing/heatpumpDevice.xml

```diff
@@ -0,0 +1,96 @@
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="melcloudaccount" />
+		</supported-bridge-type-refs>
+		<label>Heatpump Device</label>
+		<description>Heatpump device</description>
+
+		<channels>
+			<channel id="power" typeId="power-channel" />
+			<channel id="tankWaterTemperature" typeId="tankWaterTemperature-channel" />
+			<channel id="forcedHotWaterMode" typeId="forcedHotWaterMode-channel" />
+			<channel id="roomTemperatureZone1" typeId="roomTemperatureZone1-channel" />
+			<channel id="setTemperatureZone1" typeId="setTemperatureZone1-channel" />
+			<channel id="hasPendingCommand" typeId="hasPendingCommand-channel" />
+			<channel id="offline" typeId="offline-channel" />
+			<channel id="lastCommunication" typeId="lastCommunication-channel" />
+			<channel id="nextCommunication" typeId="nextCommunication-channel" />			
+		</channels>
+
+		<config-description>
+			<parameter name="deviceID" type="integer" required="true">
+				<label>Device ID</label>
+				<description>Device ID of the Heatpump device</description>
+			</parameter>
+			<parameter name="buildingID" type="integer">
+				<label>Building ID</label>
+				<description>Building ID of the Heatpump device.</description>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" required="true">
+				<label>Polling Interval</label>
+				<description>Time interval how often poll data from MELCloud</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Channels types -->
+	<channel-type id="power-channel">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power status of device</description>
+	</channel-type>
+	<channel-type id="tankWaterTemperature-channel">
+		<item-type>Number:Temperature</item-type>
+		<label>Tank Temperature</label>
+		<description>Temperature of water i</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="forcedHotWaterMode-channel">
+		<item-type>Switch</item-type>
+		<label>Forced Hot Water Mode</label>
+		<description>If water mode is Heat Now (true) or Auto (false)</description>
+	</channel-type>
+	<channel-type id="roomTemperatureZone1-channel">
+		<item-type>Number:Temperature</item-type>
+		<label>Room Temperature Zone 1</label>
+		<description>Room temperature for zone 1</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="setTemperatureZone1-channel">
+		<item-type>Number:Temperature</item-type>
+		<label>Set Temperature Zone 1</label>
+		<description>Set temperature for zone 1</description>
+		<state min="10" max="30" step="0.5" pattern="%.1f %unit%" readOnly="false" />
+	</channel-type>
+	<channel-type id="hasPendingCommand-channel">
+		<item-type>Switch</item-type>
+		<label>Pending Command</label>
+		<description>Device has a pending command(s)</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="offline-channel">
```

Comment:
Channel-types are defined globally for your binding, you should not define multiple channel-types with the same name, most bindings use a separated `channels.xml`